Bio/SeqFeature.py

```diff
@@ -1709,7 +1709,16 @@ def _flip(self, length):
         versions of Biopython which would have given join{[5:29](-), [37:52](-)}
         and the translation would have wrongly been "EXAMPLE*SILLY" instead.
 
+        When strand is None, the parts are reversed. In principle this does not
```

Comment:
How about "When all the parts have strand None, the order of the parts is reversed..."?